openpype/hosts/houdini/plugins/create/create_pointcache.py

```diff
@@ -37,13 +34,44 @@ def create(self, subset_name, instance_data, pre_create_data):
         }
 
         if self.selected_nodes:
-            parms["sop_path"] = self.selected_nodes[0].path()
+            selected_node = self.selected_nodes[0]
+
+            # Although Houdini allows ObjNode path on `sop_path` for the
+            # the ROP node we prefer it set to the SopNode path explicitly
+
+            # Allow sop level paths (e.g. /obj/geo1/box1)
+            if isinstance(selected_node, hou.SopNode):
+                parms["sop_path"] = selected_node.path()
+                self.log.debug(
+                   "Valid SopNode selection, 'SOP Path' in ROP will be set to '%s'."
+                   % selected_node.path()
+                )
+
+            # Allow object level paths to Geometry nodes (e.g. /obj/geo1)
+            # but do not allow other object level nodes types like cameras, etc.
+            elif isinstance(selected_node, hou.ObjNode) and \
+                    selected_node.type().name() in ["geo"]:
```

Comment:
Could we do that same 'less restrictive' approach to `hou.ObjNode`? :)

Reply:
I thought about it, and it will lead to some cases like 
Obj subnetwork (no sops inside) 
Obj dopnetwork (dop output will satisfy the check) 

Which will be a little bit off 🤷‍♂️